modules/monitoring/manifests/checks/aws_iam_key.pp

```diff
@@ -0,0 +1,47 @@
+# == Class: monitoring::checks::aws_iam_key
+#
+# Single Nagios alert for iam key rotations being required in the environment
+#
+# === Parameters
+#
+# [*region*]
+#   Which SES region should be checked ['us-east-1','eu-west-1']
+#
+# [*access_key*]
+#   AWS access key to use
+#
+# [*secret_key*]
+#   AWS secret key to use
+#
+# [*enabled*]
+#   Should we enable the monitoring check?
+#
+class monitoring::checks::aws_iam_key (
+  $region = undef,
+  $access_key = undef,
+  $secret_key = undef,
+  $enabled = true,
+) {
+  if $enabled and $::aws_migration {
+    $max_age = 365
```

Comment:
There's a greater possibility for confusion with the value here, and a default in the script. Could the value just be configured through changing the script?

Reply:
I wasn't sure the best way of doing this. I want to gradually decrease the max age as there are currently lots of keys that need rotation.

Reply:
It'll work as is, this was just a minor though I had. So this isn't something that I think needs changing now.